streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -1274,6 +1274,7 @@ public void shouldThrowExceptionIfApplicationServerConfigIsNotHostPortPair() {
 
     @Test
     public void shouldThrowExceptionIfApplicationServerConfigPortIsNotAnInteger() {
+        createDefaultMockTaskManager();
```

Comment:
Why this change?

Reply:
If we don't configure the TaskManager then we'll throw (the wrong kind of) exception during `configure`. Previously this test just happened to pass because we processed the user endpoint before we processed the TaskManager, so we never got to the TaskManager